river/metrics/__init__.py

```diff
@@ -26,6 +26,7 @@
     RegressionMetric,
     WrapperMetric,
 )
+from .base_internal_clustering import InternalClusteringMetrics
```

Comment:
We might come up with a shortened name for that. Besides, the name should be in singular form. Some options discussed in the meeting @hoanganhngo610, @jacobmontiel, and I had:

- IntClustMetric
- InternalClusMetric
- IClusMetric
- InterClusMetric (this one might be misleading)
- ... other ideas?

What do you think, @MaxHalford?

Reply:
I vote for `InternalClusMetric `

Reply:
I would also vote `InternalClusMetric`. In this case, I would opt to use this name first before we conclude on any further changes!